tempesta_fw/sock_srv.c

```diff
@@ -539,6 +558,7 @@ tfw_srv_conn_alloc(void)
 	INIT_LIST_HEAD(&srv_conn->fwd_queue);
 	INIT_LIST_HEAD(&srv_conn->nip_queue);
 	spin_lock_init(&srv_conn->fwd_qlock);
+	atomic_set(&srv_conn->refcnt, TFW_CONN_DEATHCNT);
```

Comment:
Need to comment why the refcounter is initialised with this value. Some thing like "Force releasing of taken server's reference counter on connection removing"

Reply:
Done.